### DISCUSSION TITLE
Best Friends and properties

### DISCUSSION THREAD

**Discussion by MustacheMan2** [DarkflameUniverse/DarkflameServer#1097] [General]
https://legouniverse.fandom.com/wiki/Friends_List?so=search

On the wiki, it states that Best Friends can visit other Best Friend's properties while they are building on them. I know the wiki is often inaccurate, so I want to find out how true this is. Because there is no video or picture evidence to the best of my ability, the only way to find out is through packet captures (if any were caught) or people who tried this out during live. Currently in DLU, if someone is present on a property while the player begins building, the property is auto set to private and the other player is removed from the instance. If this simply isn't possible, I'll edit the wiki page because I'm tired of seeing it if it isn't even accurate.

On a separate topic, wasn't a Best Friends whitelist implemented? I recall trying it and it wasn't any different than normal chat.

**Comment by EmosewaMC** [comment]
its just unimplemented right now - not very high priority.  We do have a whitelist but there are some issues with how the data is stored so we dont have the worlds largest swear word list unencrypted in a LEGO related repository.

**Reply by DarwinAnim8or** [reply]
What emo said; but to give some backstory:
DLU didn't really have any concrete plans for implementing best friends, since it'd require validating that both are adults (or have parental approval, but that required a whole different setup with parent/kid accounts), so it wasn't really a priority for us.

So when building was being implemented, it made more sense to just kick people that were on the property and set it to private (since the property would then be able to be re-inserted to the moderation queue when done)

As for live-accuracy:
As far as I know, there are no packet captures of this feature. However, there is an LU trailer that talks about it being a feature, and I think there is some anecdotal evidence from some LU community members, but that's about it.

**Reply by EmosewaMC** [reply]
from locale.xml
..."Your Best Friends can even watch you build!"...
So I think at the very least it was either planned, or was implemented but there is just little or no footage of it happening
